detect_parking_spot: 
  {
    default_speed_in_parking_zone: 1.5,
    max_distance_to_free_place: 0.4,
    speed_when_found_place: 0.8,
    length_of_parking_area: 5.5,

    point_max_x: 2,
    point_max_y: 0.2,
    point_min_x: 0.01,
    point_min_y: -0.3
  }